docs/UserGuide.md

```diff
@@ -125,52 +152,106 @@ e.g. <code>find client n/Al</code> can show clients with names like Alice and Al
 
 <div markdown="block" class="alert alert-primary">
 
 **:bookmark: Note:**<br>
 
-* Each order is associated to one time slot. Time slots are 1 hour 30 minute blocks starting from 0000 hrs. <br>
-  e.g. 1st time slot: 0000hrs to 0130hrs, 2nd time slot: 0130hrs to 0300hrs, etc.
+* `c/CLIENT_INDEX` will copy over the details of the client at the given index into the order.
+
+* `cn/CLIENT_NAME`, `cp/CLIENT_PHONE`, `ca/CLIENT_ADDRESS` will override any details copied over by `c/CLIENT_INDEX`
+
+* If `c/CLIENT_INDEX` is not specified all of `cn/CLIENT_NAME`, `cp/CLIENT_PHONE`, `ca/CLIENT_ADDRESS` must be
+  specified.
+
+* If `cn/CLIENT_NAME`, `cp/CLIENT_PHONE`, `ca/CLIENT_ADDRESS` are all specified, `c/CLIENT_INDEX` does not need to
+  be specified.
+
+* Order quantity and recipe ingredients are optional. Order quantity will be set to 1 if not specified.
 
-* When a order is added, the nearest time slot to the specified start time is taken as the order's time slot. <br>
+* Quantity of ingredients in the inventory will be decreased by the amount specified after `ri/` if it exists in the
+  inventory.
 
-* <code>p/PHONE_NUMBER</code> is the phone number of the client.
+* All orders will be uncompleted upon addition.
 
-* <code>d/DATE</code> and <code>s/START_TIME</code> represents the order date and time. They must follow the format specified [above](#features).
+* `od/DEADLINE` represents the order deadline date and time. They must follow the format specified [above](#features).
 
 * Please refer to the examples below.
 
 </div>
 
 **Examples:**
-* `add order p/89653101 d/12-12-2021 s/1030` Adds a order to the time slot nearest to 1030hrs on 12 December 2021.
+Suppose the first client in the list has the following details
+* Name: John Doe
+* Phone: 98765432
+* Address: Happy Funland Street 12
+* Email: johndoe12@gmail.com
 
-### Deleting a order: `delete order`
+* `add-o c/1 rn/Chicken Rice ri/Chicken-1-whole Rice-200-grams rp/3 od/15-11-2021 1830 oq/1` Adds an order to the
+  application where the client's name, phone and address matches the first client in the list shown above. The
+  order's recipe name and price will be chicken rice and $3 respectively. The quantity of chicken and rice will
+  decrease by 1 whole and 200 grams respectively if it exists in the inventory. The order will be scheduled to be
+  delivered by 15 November 2021 at 1830 hrs.
 
-Deletes a order from the application.
+* `add-o cn/Alex cp/98765432 ca/Hogwarts Blk 68 rn/Chicken Rice ri/Chicken-1-whole Rice-200-grams rp/3 od/15-12-2021
+  1630 oq/1` Adds an order to the
+  application where the client's name, phone and address are Alex, 98765432 and Hogwarts Blk 68 respectively. The
+  order's recipe name and price will be chicken rice and $3 respectively. The quantity of chicken and rice will
+  decrease by 1 whole and 200 grams respectively if it exists in the inventory. The order will be scheduled to be
```

Comment:
Can specify 1 here also.
Maybe can change 'scheduled to be delivered by' to 'scheduled for delivery by' or just 'delivered by'.
```suggestion
  decrease by 1 whole and 200 grams respectively if it exists in the inventory. The order of 1 chicken rice will be scheduled to be
```